.circleci/config.yml

```diff
@@ -40,7 +40,7 @@ jobs:
             cd libs
             rm -rf node_modules/web3-providers-ws/node_modules/websocket/.git/
             source ~/.bash_profile
-            nvm install 10.16
+            nvm install 14
```

Comment:
do we want to specify minor - patch version as well?

Reply:
agreed, going to go with the stable one